### PR TITLE
IM, JR | Onboarding day 2 + 3

### LINKED ISSUE #52: Onboarding screens for Day 2 and 3
Put some text for the message screens users see at the beginning of the day [WIP]

### PR DESCRIPTION
**Pull request by imanmoreira**
- Adding onboarding screens for days 2 and 3
- Moving day 1 screens to a new folder

### CODE REVIEW

**Review by zefwang — APPROVED** [review]
LGTM, must have been fun resolving merge conflicts.

File: utils/TimeUtils.ts [hunk 1]
```diff
@@ -19,8 +19,18 @@ export const getCurrentDate = () => {
...
+      day1.getMonth(),
+      day1.getDate(),
+      0
+    ).toISOString() ===
+    new Date(
+      day2.getFullYear(),
+      day2.getMonth(),
+      day2.getDate(),
+      0
+    ).toISOString()
```
Comment (dfarooq610) [inline utils/TimeUtils.ts]: Why do I feel like I approved another PR with these same changes just now

File: screens/OnboardingScreen/index.tsx [hunk 2]
```diff
@@ -1,15 +1,35 @@
...
+interface OnboardingScreenProps {
+  views: ReactElement[];
+}
+
+export const Day1Screens = [
+  <Intro1 key={2235} />,
+  <Intro2 key={22342} />,
+  <Intro3 key={23262} />,
+  <Intro4 key={7622} />,
+  <Intro5 key={87922} />,
```
Comment (dfarooq610) [inline screens/OnboardingScreen/index.tsx:24]: Are these just random numbers or are they significant

**Review by dfarooq610 — APPROVED** [review]
Lopsided Giraffes Terrify Me

File: utils/TimeUtils.ts (hunk 1 above)
Comment (jrosen081) [reply utils/TimeUtils.ts]: Oops, we had to do this to make it work. We will remove this one here lol.

File: screens/OnboardingScreen/index.tsx (hunk 2 above)
Comment (jrosen081) [reply screens/OnboardingScreen/index.tsx:24]: Super random. We just felt like 1-5 are soooooooooo boring.
